pushflatpakscript/requirements/base.in

```diff
@@ -1,4 +1 @@
-aiohttp
-pygobject
```

Comment:
I'm confused, where is this installed from then via vendoring solution?
Later edit: Nvm, found it, the `flat-manager.in` file that you're definingn below.